### PR TITLE
Homework 4

### PR DESCRIPTION
**Pull request by Volodya227**

### CODE REVIEW

File: src/com/company/Array.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+import java.util.Arrays;
+import java.util.Random;
+
+
+public class Array {
+    public int[]arr;// = new int[10];
+
+    public void print() {
+        if (true) {
```
Comment (ghost-lv) [inline src/com/company/Array.java]: what 4?

File: src/com/company/Array.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+                    }
+                }
+            }
+        }
+    }
+
+    public Array() {
+    }
+    public void spam(int len,int max){
+        this.arr =new int[len];
```
Comment (ghost-lv) [inline src/com/company/Array.java]: dont recreate, use existing

File: src/com/company/Array.java [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            }
+        }else if(anction=="guys") {
+                for (var i = 0; i < arr.length; i++) {
+                    if (i % 2 == 1) {
+                        System.out.print(arr[i]+"  ");
+                    }
+                }
+            }
+
+        System.out.println("exit");
```
Comment (ghost-lv) [inline src/com/company/Array.java]: ???

File: src/com/company/Array.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    }
+    public void r(String anction,int number){
```
Comment (Volodymyr95) [inline src/com/company/Array.java]: What the name of method?

File: src/com/company/Array.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
+package com.company;
+
+import java.util.Arrays;
+import java.util.Random;
+
+
+public class Array {
```
Comment (Volodymyr95) [inline src/com/company/Array.java]: Not all methods implemented

**Pull request merged**